packages/hash/frontend/src/pages/[accountId]/[pageEntityId].page.tsx

```diff
@@ -64,7 +65,7 @@ export const Page: React.VFC<PageProps> = ({ blocksMeta }) => {
 
   // entityId is the consistent identifier for pages (across all versions)
   const pageEntityId = router.query.pageEntityId as string;
-  const accountId = router.query.accountId as string;
+  const accountId = useCurrentWorkspaceContext().accountId!;
```

Comment:
What else do we want to include in `useCurrentWorkspaceContext`? I wonder if we can have a hook that just gives us account id (e.g. `useDerivedAccountIdFromUrl()`). We can make the hook throw if it is used on a wrong page the result will be always a string. WDYT?

I have a WIP PR that introduces slugs, we can probably do it there:  https://github.com/hashintel/hash/pull/456

Reply:
Maybe we can include more [info](https://github.com/hashintel/hash/blob/main/packages/hash/frontend/src/components/layout/PageSidebar/WorkspaceSwitcher.tsx#L36-L52) about the workspace like the workspace name, `isUserWorkspace` (if current workspace is the user's workspace), total members of the workspace (if the workspace belongs to an org). Although I don't see where these info will be used asides the Sidebar and maybe workspace homepage. 
`useDerivedAccountIdFromUrl` sounds like a better idea, especially since we wouldn't need `!`. We should probably use that

Reply:
Getting workspace info via a hook would be a good thing too. I guess we can go for: `useDerivedAccountIdFromUrl` (just returns a string) and `useWorkspaceInfo()` (returns an object with more info).